### PR TITLE
Updates to Onboarding and try experience

### PR DESCRIPTION
**Pull request by npentrel**
Made some small updates based on this page: https://www.viam.com/resources/try-viam . Let me know what you think. I think it's better than what we currently have... We may need to see where to link the update viam-server page from the installation section or whether that should be in the manage page. I'll make a ticket for that.

### CODE REVIEW

File: docs/try-viam/try-viam-tutorial.md [hunk 1]
```diff
@@ -220,10 +221,10 @@ You can [copy this `JSON` config between rental rovers](/try-viam/faq/#how-can-i
 
 ## Next steps
 
+If you have questions, check out our <a href="/try-viam/faq">FAQ</a> or join our <a href="https://discord.gg/viam">Discord Community</a> ask questions and meet other people working on robots.
```
Comment (andf-viam) [inline docs/try-viam/try-viam-tutorial.md]:
```suggestion
If you have questions, check out our <a href="/try-viam/faq">FAQ</a> or join our <a href="https://discord.gg/viam">Discord Community</a>, where you can ask questions and meet other people working on robots.
```

File: docs/try-viam/_index.md [hunk 2]
```diff
@@ -5,32 +5,48 @@ childTitleEndOverwrite: "Try Viam"
...
-Viam also exposes a {{< glossary_tooltip term_id="grpc" text="gRPC" >}} [API for robot controls](https://github.com/viamrobotics/api).
-
-Both the API and the SDKs support {{< glossary_tooltip term_id="webrtc" >}}.
-The SDKs provide a wrapper around the `viam-server` [gRPC](https://grpc.io/) API and streamline connection, authentication, and encryption against a server.
+{{< cards >}}
+{{% manualcard %}}
+{{<imgproc src="/try-viam/try-viam-1.svg" class="fill" declaredimensions=true alt="ALT">}}
+<br>
+<br>
+<h4>Click on <a href="https://app.viam.com/try">TRY</a> in Viam</h4>
```
Comment (andf-viam) [inline docs/try-viam/_index.md:25]: Recommend keeping header text unlinked.

**Review by andf-viam — COMMENTED** [review]
Great idea using the marketing images, they look great! But I hesitate to present them and the TRY steps 1-2-3 as cards that don't accept user clicks and don't represent a docs page themselves -- at odds with how these cards work everywhere else in the site. Especially noticeable with the Next step cards just below, which function as normal for the rest of the site (making the 1-2-3 TRY step cards actually seem broken at first, for example. I tried to click the images a bunch before realizing they weren't links like everywhere else).

Could we retain the great images, but not use the cards approach here? I am a huge fan of moving away from the screenshot of the TRY UI that makes it seem like we can click "Try my rover" in the image -- awesome -- but introducing that content as cards instead somewhat replicates that issue. Ping me if you want to brainstorm alternatives here! Maybe a table: small images in column 1, text for each in column 2, then make table borders invis. Or the CSS left align for images with normal paragraph text.

File: docs/try-viam/try-viam-tutorial.md [hunk 3]
```diff
@@ -220,10 +221,10 @@ You can [copy this `JSON` config between rental rovers](/try-viam/faq/#how-can-i
 
 ## Next steps
 
+If you have questions, check out our [FAQ](/try-viam/faq/) or join our [Discord Community](https://discord.gg/viam) ask questions and meet other people working on robots.
```
Comment (npentrel) [inline docs/try-viam/try-viam-tutorial.md:224]:
```suggestion
If you have questions, check out our [FAQ](/try-viam/faq/) or join our [Discord Community](https://discord.gg/viam), where you can ask questions and meet other people working on robots.
```

File: docs/try-viam/_index.md [hunk 4]
```diff
@@ -5,32 +5,45 @@ childTitleEndOverwrite: "Try Viam"
...
+      <b>2. Reserve your slot</b>
+      <p>If no one’s using a Viam Rover, you’ll take over immediately.
+Otherwise, you’ll see an estimated time for the next slot, and we’ll send you an email when it’s your turn.
+See <a href="/try-viam/reserve-a-rover/">detailed instructions</a>.</p>
+    </td>
+  </tr>
+  <tr>
+    <td>{{<imgproc src="/try-viam/try-viam-3.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="ALT">}}
+      <b>3. Get started with Viam</b>
+      <p>Try a Viam Rover in our robotics lab. <a href="/try-viam/try-viam-tutorial/">Control</a>, <a href="/try-viam/try-viam-tutorial/#base-control">teleoperate</a>, or <a href="/try-viam/tutorials/">program</a> the rover to see how you can build a robot with Viam. You can also try services like <a href="/tutorials/services/try-viam-color-detection/">computer vision.</a></p>
```
Comment (andf-viam) [inline docs/try-viam/_index.md:38]:
```suggestion
      <p>Try a Viam Rover in our robotics lab. <a href="/try-viam/try-viam-tutorial/">Control</a>, <a href="/try-viam/try-viam-tutorial/#base-control">drive</a>, or <a href="/tutorials/get-started/try-viam-sdk/">program</a> the rover to see how you can build a robot with Viam. You can also try services like <a href="/tutorials/services/try-viam-color-detection/">computer vision.</a></p>
```

opt: replacing uncommon `teleoperate` with more common `drive` (word also present in target linked section), and switching `program` to the Try SDK tutorial.

**Review by andf-viam — APPROVED** [review]
LGTM! This looks fantastic! One tiny optional language suggestion.

**Pull request merged**